client/my-sites/stats/stats-subscribers-highlight-section/index.tsx

```diff
@@ -7,24 +7,40 @@ function useSubscriberHighlights( siteId: number | null ) {
 	const translate = useTranslate();
 
 	const { data: subscribersTotals, isLoading, isError } = useSubscribersTotalsQueries( siteId );
+	// TODO: follow up on this once we have a way to determine if a site has paid newsletter set up.
+	// issue: https://github.com/Automattic/wp-calypso/issues/80609
+	const hasPaidNewsletter = true;
```

Comment:
Should we set the default value of whether paid newsletter to `true`?

> Until they configure paid subscriptions, there’s little point to show paid subscribers to them.

The original intention of https://github.com/Automattic/wp-calypso/issues/80609 seems to hide the `Paid subscribers` card by default.

Reply:
Good question! @dognose24 

The reason I set it to `true` is to make as little change as possible, so that the newsletter customer would still be able to find the `Paid subscribers` card which makes sense to them. Otherwise, I wonder where they could see the data. Also the card is already shown for everyone, so there's actually not much changed.

When we get to know whether a site has paid newsletter set up or not, we can then show the cards properly.

Does that make sense?

Reply:
Thanks for the explanation. LGTM! 👍🏼